playground/ci/find_and_validate_beam_examples.py

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from ci.find_beam_examples import find_examples
+from ci.verify_beam_example import verify_example
+
+
+def main():
+    root_dir = os.getenv("BEAM_ROOT_DIR")
```

Comment:
Is it for sure exists?

Reply:
We need this variable. As I understand we could add this variable before the CI/CD step. Is it right @snkalinin?